TextToSpeech/impl/TTSSpeaker.cpp

```diff
@@ -118,6 +118,25 @@ bool TTSConfiguration::isValid() {
     return true;
 }
 
+#if defined(PLATFORM_REALTEK)
+static void cb_new_pad (GstElement *element, GstPad *pad, gpointer data)
+{
+    gchar *name;
+    GstElement *other = (GstElement *)data;
+    name = gst_pad_get_name (pad);
+    TTSLOG_INFO ("[cb] A new pad %s was created for %s\n", name, gst_element_get_name(element));
+    g_free (name);
+    TTSLOG_INFO ("element %s will be linked to %s\n",
+            gst_element_get_name(element),
```

Comment:
should we not release memory allocated for the name as well?

Reply:
Will check and update